parlai/agents/hugging_face/t5.py

```diff
@@ -149,6 +149,15 @@ def build_dictionary(self):
         """
         return T5DictionaryAgent(self.opt)
 
+    @staticmethod
+    def dictionary_class():
+        """
+        Return the dictionary class that this agent expects to use.
+
+        Can be overridden if a more complex dictionary is required.
```

Comment:
if overriden then do we need to change L150 to be overrideable?

Reply:
ahh i had just copied from the original PR, that docstring is out of date let me fix it